### PR TITLE
Fix transmute mess

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Part of gtk-rs/gir#902

cc @EPashkin @sdroege

### CODE REVIEW

File: src/signal.rs [hunk 1]
```diff
@@ -94,7 +95,7 @@ mod editable {
                 connect_raw(
                     self.to_glib_none().0 as *mut _,
                     b"changed\0".as_ptr() as *mut _,
-                    Some(*(&trampoline::<Self, F> as *const _ as *const _)),
+                    Some(transmute(trampoline::<Self, F> as usize)),
```
Comment (sdroege) [inline src/signal.rs:98]: Please put the correct types in all the manual code :) `transmute::<_, unsafe extern "C" fn ...>(trampoline as *const ())` etc

Comment (GuillaumeGomez) [reply src/signal.rs:98]: Lazyness overload. :3

Comment (sdroege) [reply src/signal.rs:98]: This can be fixed with sed btw, no need to do that manually

**Comment by sdroege** [comment]
@GuillaumeGomez All green except for the usual macos

**Pull request merged**